js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/listGraphs/useIsOneShardDb.tsx

```diff
@@ -0,0 +1,19 @@
+import useSWR from "swr";
+import { getCurrentDB } from "../../../utils/arangoClient";
+
+const useCurrentDbProperties = () => {
+  const currentDbName = window.frontendConfig.db;
+  const { data } = useSWR(`/${currentDbName}/properties`, () => {
+    const currentDb = getCurrentDB();
+    return currentDb.request({ method: "GET", path: "/_api/database/current" });
+  });
+  return data?.body.result;
+};
+export const useIsOneShardDb = (): boolean => {
+  const currentDbProperties = useCurrentDbProperties();
+  return (
+    currentDbProperties?.sharding === "single" ||
```

Comment:
The `|| false` is redundant:

```suggestion
currentDbProperties?.sharding === "single" || window.frontendConfig.forceOneShard
```